### PR TITLE
Puppeteer firefox to codeceptjs

### PR DESCRIPTION
**Pull request by ngadiyak**
Adding support of Puppeteer-firefox to Codeceptjs

https://github.com/GoogleChrome/puppeteer/tree/master/experimental/puppeteer-firefox

### CODE REVIEW

**Comment by DavertMik** [comment]
:open_mouth: Wow! Does it really work?

**Comment by ngadiyak** [comment]
i tried it locally (including parallel launch puppeteer-chrome and puppeteer-firefox) and it worked, but i didn't try all of API methods. this is experimental feature, @DavertMik you can try it locally too )

**Comment by DavertMik** [comment]
According to their docs 90% of all methods already implemented. So I think we should pick it up. 
I will try to provide some suggestions for config API and I will merge it.

File: lib/helper/Puppeteer.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ const consoleLogStore = new Console();
  *
  * This helper should be configured in codecept.json or codecept.conf.js
  *
+ * * `browser`: (optional, default: chrome) - select browser
```
Comment (DavertMik) [inline lib/helper/Puppeteer.js]: - select browser: chrome or firefox (experiemental, requires `puppeteer-firefox` package)

Comment (DavertMik) [reply lib/helper/Puppeteer.js]: Btw, I don't see where default value for `browser` is acutally set

**Comment by DavertMik** [comment]
@ngadiyak Please add default value for `browser` and I am merging! :+1:

**Comment by ngadiyak** [comment]
https://github.com/ngadiyak/CodeceptJS/blob/dbc2f836cadef27fa4e98608418ebc807054d1c9/lib/helper/Puppeteer.js#L145

**Comment by DavertMik** [comment]
I was thinking to put `browser: 'chrome'` somewhere here https://github.com/ngadiyak/CodeceptJS/blob/dbc2f836cadef27fa4e98608418ebc807054d1c9/lib/helper/Puppeteer.js#L156

**Comment by ngadiyak** [comment]
@DavertMik done

**Comment by DavertMik** [comment]
Ok, I will merge it when I will be releasing a new version

**Pull request merged**